frontend/app/views/fragments/form/feedback.scala.html

```diff
@@ -64,12 +64,14 @@ <h2 class="fieldset__headline">About you</h2>
             @fragments.form.errorMessage("Please enter your name")
         </div>
 
+        @* email regex taken from http://www.w3.org/TR/html-markup/input.email.html#input.email.attrs.value.single *@
         <div class="form-field">
             <label class="label" for="email">Email address</label>
             <input name="email"
                    id="email"
                    class="input-text"
                    type="email"
+                   pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$"
```

Comment:
The pattern is used in the JS, not as part of the native browser validation. So I think @feedmypixel  is using the pattern as a regex in the JS. Arguably this should be defined in the JS in one location though, rather than per-element.